scripts/LargeScaleStructures/geometry_writer.py

```diff
@@ -1,10 +1,10 @@
-#pylint: disable=invalid-name
+# pylint: disable=invalid-name,unused-variable, too-many-arguments
```

Comment:
For the `unused-variable` they should be removed or they can be replaced with the `_`.
For the `too-many-arguments` warning I'd rather keep it there if it's not possible to really fix it.